webpack/scenes/ContentViews/__tests__/contentViewPage.test.js

```diff
@@ -1,16 +1,18 @@
+/* eslint-disable no-useless-escape */
```

Comment:
This file took between 11 and 21 seconds to run on my system.  Any idea why it's slow?  Is there any way it can be sped up?